modules/op-configuring-scc-namespace.adoc

```diff
@@ -0,0 +1,23 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/configuring-security-context-for-pods.adoc
+:_content-type: PROCEDURE
+[id="op-configuring-scc-namespace_{context}"]
+= Configuring the SCC for pods in a namespace
+
+You can configure the security context constraint (SCC) for all pods that {pipelines-shortname} creates for pipeline runs and task runs that you create in a particular namespace. This SCC must not be less restrictive than the maximum SCC that you configured using the `TektonConfig` CR, in the `spec.platforms.openshift.scc.maxAllowed` spec.
+
+.Procedure
+
+* Set the `operator.tekton.dev/scc` annotation for the namespace to the name of the SCC.
+
```

Comment:
I'd add a `+` on this line so that it's clearer that the example is directly related to the step.
If possible, I would also suggest adding info about how to open this manifest for editing.

Reply:
Unfortunately can't cycle back to engineering at this stage; annotations are already referenced several times in the doc set, so this is realy an existing issue and I created a backlog Jira for it at https://issues.redhat.com/browse/RHDEVDOCS-5634